build/gulpfile.vscode.js

```diff
@@ -500,9 +499,13 @@ function packageTask(platform, arch, opts) {
 const buildRoot = path.dirname(root);
 
 // {{SQL CARBON EDIT}}
-gulp.task('vscode-win32-x64-azurecore', ['optimize-vscode'], packageAzureCoreTask('win32', 'x64'));
-gulp.task('vscode-darwin-azurecore', ['optimize-vscode'], packageAzureCoreTask('darwin'));
-gulp.task('vscode-linux-x64-azurecore', ['optimize-vscode'], packageAzureCoreTask('linux', 'x64'));
+gulp.task('vscode-win32-x64-azurecore', ['optimize-vscode'], packageExtensionTask('azurecore', 'win32', 'x64'));
+gulp.task('vscode-darwin-azurecore', ['optimize-vscode'], packageExtensionTask('azurecore', 'darwin'));
+gulp.task('vscode-linux-x64-azurecore', ['optimize-vscode'], packageExtensionTask('azurecore', 'linux', 'x64'));
+
+gulp.task('vscode-win32-x64-mssql', ['optimize-vscode'], packageExtensionTask('mssql', 'win32', 'x64'));
```

Comment:
maybe not the most scalable way to solve the problem, but probably good enough.  mssql and azurecore are the two most problematic extensions, and we should avoid creating additional "heavy weight" extensions like these that have 1000s of files.